src/components/subscribe/subscribe.scss

```diff
@@ -12,14 +12,20 @@
   position: relative;
 }
 
+.c-subscribe__heading {
+  margin-block-end: size.$rhythm-default-rem;
```

Comment:
I removed the margins from the controls and the form because they were messing up the layout when there's no heading.

I moved the margin here to the heading instead, and changed it to rem units so the layout wouldn't change due to the heading's larger font size.